eng/Packages.props

```diff
@@ -4,7 +4,10 @@
     <DotNetAssemblyVersion>3.1.3</DotNetAssemblyVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Update="Azure.Core" Version="1.0.2" /> 
+    <!-- Note: These libraries can be loaded by Arcade SDK clients, which may already have a
+         newer copy of this package loaded. When updating one, be aware of the other to prevent runtime issues
+         See: https://github.com/dotnet/arcade/blob/main/eng/Versions.props -->
+    <PackageReference Update="Azure.Core" Version="1.19.0" />
```

Comment:
Hopefully this doesn't break the services?

Reply:
the only reference to the package in the repo seems to be in Microsoft.DotNet.Maestro.Client, so I think we'll be OK but I'll be on point to eat those words as well if needed.